.github/workflows/swift-toolchain.yml

```diff
@@ -1071,8 +1076,8 @@ jobs:
           bin-dir: ${{ github.workspace }}/BinaryCache/1
           install-dir: ${{ github.workspace }}/BuildRoot/Library/Developer/Toolchains/${{ inputs.swift_version }}+${{ matrix.variant }}/usr
           swift-sdk-path: ${{ steps.setup-context.outputs.sdkroot }}
-          msvc-compilers: '@("C", "CXX")'
-          pinned-compilers: '@("Swift")'
+          msvc-compilers: ${{ inputs.build_compilers_with_llvm && '@()' || '@("C", "CXX")' }}
```

Comment:
If we use clang-cl, we should consistently use that throughout the entire build I think

Reply:
Yes, if we are building with clang, that should be using lld-link universally as well.